content/api/authentication/systemuserapi/clientdelegation/_index.en.md

```
@@ -0,0 +1,321 @@
+Removes a client from the system user
+
+### Endpoint
+DELETE authentication/api/v1/enduser/systemuser/clients/
+
+### Scopes
+Id porten token with scope <mark>altinn:clientdelegations.read altinn:clientdelegations.write</mark>
+
+### Content types
+application/json
+
+## Query Parameters
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)